coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 100.0
+    "covered_percent": 73.33
```

Comment:
I actually bet that you just need to run rspec again, I know you didn't drop the coverage to the 70's